src/collections/safe_list.zig

```diff
@@ -335,7 +335,7 @@ pub fn SafeMultiList(comptime T: type) type {
                 const cur_items = base.items(cur_field);
 
                 new_ptrs[i] = if (cur_items.len == 0)
-                    @as([*]u8, @ptrFromInt(@alignOf(u8)))
+                    @as([*]u8, @ptrFromInt(@alignOf(std.meta.Elem(@TypeOf(cur_items)))))
```

Comment:
Does something like this work?
```suggestion
                    @as([*]u8, @ptrFromInt(@alignOf(T)))
```